lib/src/client.dart

```diff
@@ -122,19 +122,12 @@ class Client {
     if (response.statusCode == 201) {
       _setSession(response);
 
-      return response.headers[HttpHeaders.locationHeader];
+      return {
+        "success": {"orderURL": response.headers[HttpHeaders.locationHeader]}
+      };
```

Comment:
Not sure how much I like the `"success"` wrapper. But I feel like I need to signal success. Maybe by lack of `errors` field means success?